geomstats/geometry/general_linear.py

```diff
@@ -114,9 +114,9 @@ def orbit(cls, point, base_point=None):
 
         Will work when expm gets properly 4-D vectorized.
```

Comment:
"Will work when..."
Sorry, I had missed this comment in a previous code review.

Could you update this comment into a TODO(ninamiolane)? Thanks!